src/Smalot/PdfParser/PDFObject.php

```diff
@@ -191,7 +191,7 @@ public function getSectionsText(?string $content): array
         $textCleaned = $this->cleanContent($content, '_');
 
         // Extract text blocks.
-        if (preg_match_all('/(\sQ)?\s+BT[\s|\(|\[]+(.*?)\s*ET(\sq)?/s', $textCleaned, $matches, \PREG_OFFSET_CAPTURE)) {
+        if (preg_match_all('/(.*?)\s+BT[\s|\(|\[]+(.*?)\s*ET(?=\s|$)?/s', $textCleaned, $matches, \PREG_OFFSET_CAPTURE)) {
```

Comment:
```suggestion
        $regex = '/(.*?)\s+BT[\s|\(|\[]+(.*?)\s*ET(?=\s|$)?/s';
        if (preg_match_all($regex, $textCleaned, $matches, \PREG_OFFSET_CAPTURE)) {
```

Makes it easier to read.